pkgs/development/python-modules/python-chess/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "python-chess";
+  version = "1.6.1";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "chess";
+    sha256 = "sha256-lmumSDVZ3A+fOr1vryvKLhBVBGavQjvnmodlY509zTM=";
+  };
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A chess library for Python, with move generation, move validation, and support for common formats.";
```

```suggestion
    description = "A chess library for Python, with move generation, move validation, and support for common formats";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

---

Fixed.